### ISSUE TITLE
Restart Option

### ISSUE THREAD

**Issue by win-lee** [s0faking/plugin.video.orftvthek#82]
Restart Option doesn't seem to work. (Live works fine though)
Any known issues with that?

**Comment by s0faking** [comment]
Yes, this issue is happening since the major jump to KODI 18 (Restart still works fine on 17). The inputstream.adaptive addon doesn't play the orf playlist files. I think this has something to do with:

https://github.com/peak3d/inputstream.adaptive/issues/131

**Comment by win-lee** [comment]
Ok, thanks for the info.
If you do or see any changes maybe you could drop me a note...
Cheers!
